sources/imperative/ReaderUtils.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+extension InputStream {
+    /// Stream for reading from stdin.
+    public static var stdin: InputStream {
+        return InputStream(fileAtPath: "/dev/stdin")!
```

Comment:
This force unwraps here. But maybe we should `fatalError` with a custom message?

I suppose it could fail on non-unix platforms or if your binary was in a chroot'd fs (like a docker container) and there's just no `/dev/stdin` mounted.